code/modules/projectiles/guns/energy/dueling.dm

```diff
@@ -192,6 +192,8 @@
 
 /obj/item/gun/energy/dueling/can_trigger_gun(mob/living/user)
 	. = ..()
+	if(!duel)
+		return
```

Comment:
This doesn't look right?

Reply:
Yeah, I'll likely do this in a seperate PR since dueling just seems to be a whole other sort of broken.